src/AzureClient/AzureClient.cs

```diff
@@ -289,8 +289,11 @@ public async Task<ExecutionResult> SetActiveTargetAsync(IChannel channel, string
             // Set the active target and load the package.
             ActiveTarget = executionTarget;
 
-            channel.Stdout($"Loading package {ActiveTarget.PackageName} and dependencies...");
-            await References.AddPackage(ActiveTarget.PackageName);
+            if (!(ActiveWorkspace is MockAzureWorkspace))
+            {
+                channel.Stdout($"Loading package {ActiveTarget.PackageName} and dependencies...");
+                await References.AddPackage(ActiveTarget.PackageName);
+            }
```

Comment:
This makes sense for now. I've opened #223 to restore the test coverage for loading provider packages here.